python/hsfs/engine/python.py

```diff
@@ -284,31 +283,11 @@ def save_dataframe(
         online_write_options,
         validation_id=None,
     ):
-        # App configuration
-        app_options = self._get_app_options(offline_write_options)
-
-        # Setup job for ingestion
-        # Configure Hopsworks ingestion job
-        print("Configuring ingestion job...")
-        fg_api = feature_group_api.FeatureGroupApi(feature_group.feature_store_id)
-        ingestion_job = fg_api.ingestion(feature_group, app_options)
-
-        # Upload dataframe into Hopsworks
-        print("Uploading Pandas dataframe...")
-        self._dataset_api.upload(feature_group, ingestion_job.data_path, dataframe)
-
-        # Launch job
-        print("Launching ingestion job...")
-        self._job_api.launch(ingestion_job.job.name)
-        print(
-            "Ingestion Job started successfully, you can follow the progress at {}".format(
-                self._get_job_url(ingestion_job.job.href)
-            )
-        )
+        self._save_online_dataframe(feature_group, dataframe, online_write_options)
```

Comment:
I am not sure if we could keep the old implementation for the 2.6 release until we implement this.
At least we should throw a NotImplementedError

However, I say we keep the old implementation otherwise the Python client will be unusable in the 2.6 release